mod_reforged/hooks/entity/tactical/humans/nomad_archer.nut

```diff
@@ -50,15 +50,11 @@
 		// }
 
 		// Reforged
```

Comment:
The `// Reforged` comment can be removed as well at this point as there's nothing below it. We can re-add the comment when/if we make changes to this entity.